src/modules/ModuleStatementHome.tsx

```diff
@@ -0,0 +1,83 @@
+import { AnimatableNumber } from "@/components/core/AnimatableNumber";
+import { RichText } from "@/components/core/RichText";
+import { Section } from "@/components/core/Section";
+import { AssetFragment } from "@/lib/contentful/fragments/AssetFragment";
+import { ComponentStatisticFragment } from "@/lib/contentful/fragments/ComponentStatisticFragment";
+import { GetModuleStatementHomeById } from "@/lib/contentful/query/GetModuleStatementHomeById";
+import { cx } from "cva";
+import { readFragment, ResultOf } from "gql.tada";
+
+export const ModuleStatementHome = ({
+  data,
+}: {
+  data: ResultOf<typeof GetModuleStatementHomeById>;
+}) => (
+  <Section className="dark py-16 dsk:pt-32 dsk:pb-20 flex flex-col items-center">
+    <h2 className="typo-display font-light text-center px-6 max-w-5xl">
+      <RichText
+        content={data.moduleStatementHome?.headline}
+        variant="title"
+        spansOnly
+      />
+    </h2>
+    <ul className="pt-16 dsk:pt-20 w-full items-center justify-between flex flex-col dsk:flex-row gap-5">
+      {data.moduleStatementHome?.statisticsCollection?.items.map(
+        (item, index) => {
+          const statistic = readFragment(ComponentStatisticFragment, item);
+          if (!statistic) return null;
+          return (
+            <div
```

Comment:
```suggestion
            <li
```

If you're going to use UL, let's update this to improve HTML semantics.